_includes/v21.2/sql/expressions-as-on-conflict-targets.md

```diff
@@ -0,0 +1,49 @@
+CockroachDB does not support expressions as `ON CONFLICT` targets. This means that unique {% if page.name == "expression-indexes.md" %} expression indexes {% else %} [expression indexes](expression-indexes.html) {% endif %} cannot be selected as arbiters for [`INSERT .. ON CONFLICT`](insert.html#on-conflict-clause) statements. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET experimental_enable_expression_indexes=true;
+~~~
+
+~~~
+SET
```

Comment:
You can remove the SET command and SET output - as of 21.2 expression indexes are enabled by default.

Reply:
NOICE - removed